scripts/startup/tmux_launch.sh

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash -ex
 
-# This script is designed to be run from /etc/rc.local (or similar)
-# as user $PANUSER with the PANOPTES environment variables set.
+# Start the PANOPTES software in a detached tmux (terminal multiplexer)
+# session; this enables the PANOPTES unit's owner/administrator to
+# attach to that session later, see the output and take control of all
+# the shells.
```

Comment:
Might be worth included an example of how to attach to session.

Reply:
Done, here and in README.md.